hbase-client/src/main/java/org/apache/hadoop/hbase/protobuf/ProtobufUtil.java

```diff
@@ -1773,12 +1775,15 @@ public static ServerName toServerName(final byte [] data) throws Deserialization
 
   public static RSGroupInfo toGroupInfo(RSGroupProtos.RSGroupInfo proto) {
     RSGroupInfo RSGroupInfo = new RSGroupInfo(proto.getName());
-    for(HBaseProtos.ServerName el: proto.getServersList()) {
-      RSGroupInfo.addServer(Address.fromParts(el.getHostName(), el.getPort()));
-    }
-    for(TableProtos.TableName pTableName: proto.getTablesList()) {
-      RSGroupInfo.addTable(ProtobufUtil.toTableName(pTableName));
-    }
+
+    Collection<Address> addresses = proto.getServersList().parallelStream()
+      .map(serverName -> Address.fromParts(serverName.getHostName(), serverName.getPort()))
+      .collect(Collectors.toList());
+    RSGroupInfo.addAllServers(addresses);
```

Comment:
nit: Rename the local variable name from `RSGroupInfo`  to `rsGroupInfo`

Reply:
Updated